tests/integration/tasks/test_azure_data_lake.py

```diff
@@ -8,69 +8,70 @@
     AzureDataLakeList,
     AzureDataLakeToDF,
     AzureDataLakeUpload,
+    AzureDataLakeRemove,
 )
-from viadot.tasks.azure_data_lake import AzureDataLakeRemove
+
 
 uuid_4 = uuid.uuid4()
 uuid_4_2 = uuid.uuid4()
 
-file_name = f"test_file_{uuid_4}.csv"
-file_name_2 = f"test_file_{uuid_4}.csv"
-adls_path = f"raw/supermetrics/{file_name}"
-adls_path_2 = f"raw/supermetrics/{file_name_2}"
+FILE_NAME = f"test_file_{uuid_4}.csv"
+FILE_NAME_2 = f"test_file_{uuid_4}.csv"
+ADLS_PATH = f"raw/supermetrics/{FILE_NAME}"
+ADLS_PATH_2 = f"raw/supermetrics/{FILE_NAME_2}"
 
-file_name_parquet = f"test_file_{uuid_4}.parquet"
-adls_path_parquet = f"raw/supermetrics/{file_name_parquet}"
+FILE_NAME_PARQUET = f"test_file_{uuid_4}.parquet"
+ADLS_PATH_PARQUET = f"raw/supermetrics/{FILE_NAME_PARQUET}"
 
 # TODO: add pytest-depends as download tests depend on the upload
```

Comment:
I would add comment about adding dependency between remove and upload or just add needed dependencies :)